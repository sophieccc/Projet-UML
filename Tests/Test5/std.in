1
one
111
3
5
3