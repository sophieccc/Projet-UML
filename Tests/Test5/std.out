Radius of the purified zone : 0